sao/src/view/tree.js

```diff
@@ -1080,7 +1080,7 @@
                 }
             }).done(() => {
                 if (!this.record && this.rows.length) {
-                    this.rows[0].select_row({});
+                    this.rows[0].select_row();
```

Comment:
Pour moi il est plus simple de faire

`this.rows[0].select_row(new Event({}))`